### DISCUSSION TITLE
more apparent focus by opacity or a stroke

### DISCUSSION THREAD

**Discussion by Stooovie** [ianyh/Amethyst#1486] [Ideas]
Hi, MacOS has relatively low visual differentiation of focused window. It would be great if Amethyst could either lower opacity or contrast of unfocused windows, OR draw a thin stroke/border around focused window. Possible?

**Comment by ianyh** [comment]
There is a long-standing request that is being tracked here: https://github.com/ianyh/Amethyst/issues/127

My general thought is that it's super reasonable in theory, but I'm not sure how best to implement it in practice.
